### ISSUE TITLE
Just a question about a few unavailable functions

### ISSUE THREAD

**Issue by jm314159** [Andre0512/hon#170]
I noticed that some functions, among others:
- screenDisplayStatus
- silentSleepStatus
- night_mode

are unavailable in Home Assistant from some time. Is it related to the main [case](https://github.com/Andre0512/hon/issues/147)?
I truly regret these functions are missing in HE integration.

**Comment by jfmfreire** [comment]
I have the same issue.

![Screenshot_2024-02-16-14-10-38-453_io homeassistant companion android](https://github.com/Andre0512/hon/assets/62487216/d9b9c6e1-3e60-46d3-9e73-ffde0603e285)

**Comment by animalillo** [comment]
i also found this issue, with a bunch of entities not updating and being unavailable, specially that screen display one that i use every night to be able to sleep xD

Those are my HA versions:
| Component | Version | 
| -------| -------------| 
| Core | 2024.2.2| 
| Supervisor | 2024.01.1| 
| Operating System | 11.5| 
| Frontend | 20240207.1| 

I found this on the logs, which i'm not sure if it's related:

```
Logger: homeassistant.components.climate
Source: components/climate/__init__.py:354
Integration: Climatización (documentation, issues)
First occurred: 11:20:41 (2 occurrences)
Last logged: 11:20:41

Entity None (<class 'custom_components.hon.climate.HonACClimateEntity'>) implements HVACMode(s): off, auto, cool, dry, heat, fan_only and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/Andre0512/hon/issues

**Comment by NoUsername10** [comment]
Hello.
Great that im not alone, i have the same issues, screen display is a must for sleeping :-)
Tried to restart, downgrade to v12 but that did not help.

Ver 13.0 hOn integration.
Hass:
![Home assistant version](https://github.com/Andre0512/hon/assets/71009757/43c52538-6f6f-4893-9b51-200489656772)


Enteties error:
![Haier disabled entity](https://github.com/Andre0512/hon/assets/71009757/3a5073f8-8ef1-4a50-b531-e144dacf414f)

**Comment by jm314159** [comment]
Well, I can only state that after doing some monkey tests I found the (still) working config with:
```
Core version: 2023.12.4
hOn version: 0.10.0
```
A core downgrade itself can be done in the ssh terminal with a command:
```
ha core update --version 2023.12.4
```
Installing any newer core disables hOn usability in both versions 0.10.0 and 0.13.0.

**Comment by NoUsername10** [comment]
> Well, I can only state that after doing some monkey tests I found the (still) working config with:
> 
> ```
> Core version: 2023.12.4
> hOn version: 0.10.0

I’ve had it working with hOn v12, but I think the latest HASS upgrade caused it for me.

Strange one this.

**Comment by InCrustWeTrust** [comment]
Same issue for me.

**Comment by droeloe1818** [comment]
Same problem here, 

![image](https://github.com/Andre0512/hon/assets/82944477/4e33ea30-cec7-469f-bee6-8487de295a1e)

Downgrade to version 12 solved the problem.

@jm314159 
I think you can add the label Bug and hope @Andre0512 will take a look into this.

**Comment by jm314159** [comment]
One hint comes to my mind as long as you should have missed it. Upon the integration upgrade/downgrade, remember to remove the whole content of the directory where you used to store the project (/config/custom_components/hon) including the directory __pycache__ prior to pasting a new content in there. Then, restart Home Assistant.

**Comment by Andre0512** [comment]
Hi guys, @VadymMelnychuk fixed it in Andre0512/pyhOn#23 :partying_face: 
I created a new beta release, please check it out.

**Comment by jm314159** [comment]
Confirm, after the update to 0.14.0-beta.1 all functions to operate klima are available to me.
Really fixed!
Thanks, Guys @Andre0512 and @VadymMelnychuk
